xbmc/platform/linux/network/zeroconf/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+if(AVAHI_FOUND)
```

Comment:
There is no need for this check, as this file is only processed if Avahi is found.